docs/sidebars.js

```diff
@@ -148,6 +148,7 @@ module.exports = {
           },
           collapsed: true,
           items: [
+            "development/hardware-integration/new-board",
```

Comment:
I'd like to move this down, e.g. to before "battery sensing". Everything above that is more directly related to shields/keyboards and will be relevant to more people.